### DISCUSSION TITLE
The DAT protocol will be drop totally?

### DISCUSSION THREAD

**Discussion by lumenpink** [beakerbrowser/beaker#1554] [General] [answered]
Will  the new Hyperdrive protocol supersede it?

**Comment by pfrazee** [comment, marked as answer]
Correct, we're transitioning from `dat://` to `hyper://`. A lot of the core tech is the same (same developers, etc) but it's a breaking set of ~changes~ _improvements_.

Beaker has a dat -> hyper converter which you can use to update your library.

**Reply by Foldover** [reply]
I wanted to try the conversion tool, but it merely seem to create an empty `index.json` file. I suppose that is not the expected behaviour?

**Reply by pfrazee** [reply]
@Foldover the converter currently only loads dats that were in your library. It doesn't access the network, so any dat hosted via the cli `dat` command for instance won't load. Does that answer your situation?

**Reply by Foldover** [reply]
Thanks for replying so fast! The dat was in my library, created via Beaker's browser tools, no CLI involved :). The legacy dat was created in Beaker 0.8 I believe.

**Reply by pfrazee** [reply]
Hmm sounds like a bug. There does seem to be some interest in better tooling here so I may invest a little more time in better dat:// readonly support and conversion tools. (No promises yet but we're looking into it.)
